src/parser/jobs/whm/modules/Overheal.tsx

```diff
@@ -29,6 +31,15 @@ export class Overheal extends CoreOverheal {
 			trackedHealIds: [
 				this.data.actions.LITURGY_OF_THE_BELL_ON_DAMAGE.id,
 				this.data.actions.LITURGY_OF_THE_BELL_ON_EXPIRY.id,
+				this.data.actions.LITURGY_OF_THE_BELL_ACTIVATION.id,
+			],
+		},
+		{
+			name: <Trans id="whm.overheal.afflatus.name">Afflatus Healing</Trans>,
+			color: SuggestedColors[4],
```

Comment:
Out of scope for this PR but this is not very safe. Core overheal should use an object or map and have actual keys for colours rather than relying on an opaque array index, or this should be entirely up to consuming jobs. IMO set the colour directly for now.

Reply:
I put the actual hex colors that were obscured by the array prior, not sure if that's better, but at least they're directly visible now?